angularjs-portal-frame/src/main/webapp/portal/settings/partials/settings.html

```diff
@@ -90,6 +90,16 @@ <h3 class="home-section-header">Your Options</h3>
              </h4>
              <small>Show the My Courses, Email, and Calendar example widgets</small>
          </li>
+         <li class="portlet-container-home beta-card-style">
+             <h4>
+            <span class="btn-group">
+              <label class="btn" ng-class="{'btn-success' : $storage.mobileWidgetToggle, 'btn-default' : !$storage.mobileWidgetToggle }" ng-model="$storage.mobileWidgetToggle" btn-radio="true">Show</label>
+              <label class="btn" ng-class="{'btn-success' : $storage.mobileWidgetToggle, 'btn-default' : !$storage.mobileWidgetToggle }" ng-model="$storage.mobileWidgetToggle" btn-radio="false">Hide</label>
+            </span>
+                 Widget Toggle
```

Comment:
Nitpick: how about a title of something more like "Widget Toggle in Mobile".  That is, getting Mobile into the title of this setting.